planning/behavior_path_planner/src/utils/utils.cpp

```diff
@@ -1606,6 +1606,10 @@ std::vector<geometry_msgs::msg::Point> calcBound(
       const auto polygon =
         route_handler->getLaneletMapPtr()->polygonLayer.get(std::atoi(id.c_str()));
 
+      const auto is_clockwise_polygon =
+        boost::geometry::is_valid(lanelet::utils::to2D(polygon.basicPolygon()));
+      const auto is_clockwise_iteration = is_clockwise_polygon ? is_left : !is_left;
+
```

Comment:
How about just using `boost::geometry::correct(polygon)`?
It's much easier to understand for me.

Reply:
I understand what you said but boost correct only supports basic polygon (basic~ doesn't have id). So, it can't correct polygon orientation with each point id in my understanding. This is why I didn't use boost correct.

Reply:
@takayuki5168 Do you have any other concern?